rpm/toggle-monitor/toggle-monitor.spec

```
@@ -0,0 +1,31 @@
+Name:     toggle-monitor
+Version:  1
+Release:  1
+Summary:  Script to turn monitor on and off
+License:  Public Domain
+
+Requires: xrandr speech-dispatcher-utils
+
+SOURCE0:  monitor-toggle
+SOURCE1:  10-keybindings-toggle-monitor
+
+%description
+RPM package with toggle-monitor script
+
+%prep
+
+%build
+
+%install
+install -d %{buildroot}%{_bindir}
+install -m 755 %{SOURCE0} %{buildroot}%{_bindir}/monitor-toggle
+install -d %{buildroot}%{_sysconfdir}/dconf/db/local.d
+install %{SOURCE1} %{buildroot}%{_sysconfdir}/dconf/db/local.d/10-keybindings-toggle-monitor
+
+%files
+%{_bindir}/monitor-toggle
+%{_sysconfdir}/dconf/db/local.d/10-keybindings-toggle-monitor
```

I suggest we change this because we no longer perform some local configuration, we ship it in package.
```suggestion
%{_sysconfdir}/dconf/db/distro.d/10-keybindings-toggle-monitor
```

---

Changed as requested.